postgres_lambda_strategy/README.md

```diff
@@ -0,0 +1,28 @@
+# Postgres Lambda Strategy
+
+A starter example to implement Postgres Sym Flow that uses an AWS Lambda to manage database user permissions.
+
+## Tutorial
+
+Our step-by-step Lambda [tutorial](https://docs.symops.com/docs/aws-lambda) will get you most of the way through the setup for Postgres.
+
+## Example database setup
+
+We've included a [helper module](helper) that you can use to provision a VPC and an RDS Postgres database suitable for testing the integration.
+
+Use the outputs of the helper module as input variables to finish setting up an end to end example.
+
+## Local testing
+
+You can iterate on your handler function locally by setting up a docker-compose based Postgres database and then invoking your handler function directly.
+
+1. Start the local database with [`docker-compose`](src/test/docker-compose.yaml).
```

Comment:
`compose` is now native to `docker` and they recommend using `docker compose` now, instead of `docker-compose`

https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command